progressive-cactus/0.1/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM biocontainers/biocontainers:08252016-4
+
+LABEL base.image="biocontainers:08252016-4"
+LABEL version="0.1"
+LABEL software="progressiveCactus"
+LABEL software.version="08092017"
```

Comment:
should set the tagged version here ie 0.1

Reply:
On the line 4 is set the version to 0.1. Do you mean to change the LABEL software.version to 0.1 ?

Reply:
version is the "build" number, used when modifying the Dockerfile for example but not the upstream source version. software.version must match the upstream tag here 0.1, and version would be 1

Reply:
ok dokie